GRAPHICS/DisableMoonCycleOverride.md

```diff
@@ -0,0 +1,19 @@
+---
+ns: GRAPHICS
+aliases: ["0x2BF72AD5B41AA739"]
+---
+
+## DISABLE_MOON_CYCLE_OVERRIDE
+
+```c
+// 0x2BF72AD5B41AA739 b323
+void DISABLE_MOON_CYCLE_OVERRIDE();
+```
+
+Removes any custom moon cycle overrides that have been configured with [ENABLE_MOON_CYCLE_OVERRIDE](./EnableMoonCycleOverride.md)
```

Comment:
Links to other native docs should be done by their hash `[ENABLE_MOON_CYCLE_OVERRIDE](#_0x2C328AF17210F009)`

Reply:
Fixed in latest comment.